examples/example_util.py

```diff
@@ -53,13 +53,18 @@ def show_metrics(rbm, performance):
 
 def compute_reconstructions(rbm, v_data, fit, n_recon=10, vertical=False):
     sampler = fit.DrivenSequentialMC(rbm)
-    data_state = State.from_visible(v_data, rbm)
+    data = v_data[0] if isinstance(v_data, tuple) else v_data
```

Comment:
It would be better if each function had a single purpose, and the types of the arguments are clear.  A distinct function should handle displaying class predictions.  

Separate from this PR, some of these functions are lacking docstrings.